test_requirements.txt

```diff
@@ -0,0 +1,6 @@
+git+https://github.com/ONSDigital/sdx-common.git@0.7.0
+coverage==4.4.1
+responses==0.5.1
+pep8==1.6.2
+flake8==2.6.0
+structlog==17.2.0
```

Comment:
structlog should be a runtime dependency, it's use in the code base, not just in tests. Please move to requirements.txt